source/webgl-operate.ts

```diff
@@ -13,6 +13,15 @@ export import fetch = root_fetch;
 import * as root_raymath from './raymath';
 export import ray_math = root_raymath;
 
+export { FontFace } from './fontface';
+export { FontLoader } from './fontloader';
+export { GlyphVertex, GlyphVertices } from './glyphvertices';
+export { Label } from './label';
+export { Position2DLabel } from './position2dlabel';
+export { Position3DLabel } from './position3dlabel';
```

Comment:
Within the current naming scheme, it might be better to either use 
`Texture2D` and `Texture3D` or `Position2Label` and `Position3Label` - i prefer to skip the `D` because:
* the gl-matrix vec and mat classes do not have these
* webgl (and opengl in general) uses this kind of syntax in its c-based interface which is also used in webgl, e.g., `uniform2f` or `uniform2iv` ... on the other hand there is `glTexImage2D` and `glTexImage3D` ... also .. in GLSL it is `vec2` but `sampler2D` ... so probably go for the `D`... @bwasty, @anne-gropler  what do you think?

Reply:
At some point, there will be a "projected3DLabel", as introduced in the OpenLL API examples. Would "projected3Label" still be comprehensible?

I have no problem in skipping the D.

Reply:
just talked to @scheibel - he prefers `D` and @sbusch42 i remember also prefering the `D` ... same for @bwasty  - since i'm the only one for NO-D ... i will create an issue to rename the `Texture2` and `Texture3` classes ... so no change w.r.t to the label classes.

Reply:
issue created #105